modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/web/web.js

```diff
@@ -13,23 +13,23 @@
  * HANA XS Classic Bridge for Web API
  */
 
-var dRequest = require('http/v4/request');
-var dResponse = require('http/v4/response');
-var dUpload = require('http/v4/upload');
-var session = require('xsk/session/session');
-var zip = require('io/v4/zip');
-var bytes = require("io/v4/bytes");
+const dRequest = require('http/v4/request');
+const dResponse = require('http/v4/response');
+const dUpload = require('http/v4/upload');
+const session = require('xsk/session/session');
+const zip = require('io/v4/zip');
+const bytes = require("io/v4/bytes");
 
-var WEB_UTILS = require('xsk/web/webUtils');
+const WEB_UTILS = require('xsk/web/webUtils');
 
-var SET_COOKIE_HEADER = "Set-Cookie";
+const SET_COOKIE_HEADER = "Set-Cookie";
 
-const TupelList = function (dArrayOfContents, isReadOnly) {
-  var internalDArrayOfContents = dArrayOfContents ? dArrayOfContents : [];
+exports.TupelList = function (dArrayOfContents, isReadOnly) {
+  let internalDArrayOfContents = dArrayOfContents ? dArrayOfContents instanceof Array ? dArrayOfContents : Object.values(dArrayOfContents) : [];
```

Comment:
This does not look ok. Why are you passing arrays and objects?

Reply:
We expect an array but seems we are not sure what customer application will pass. That's why we add this additional check and explicitly convert input to array.

Reply:
How can we be not sure? There is documentation about the TupelList's API in classic xsjs: https://help.sap.com/doc/3de842783af24336b6305a3c0223a369/2.0.03/en-US/$.web.TupelList.html

Also, in the example of the issue this PR fixes, it's shown how the Request API is used:
```JS
var client = new http.Client();
var request = new http.Request(http.GET, "/Orders");
request.parameters.set("$expand", "Customer,Order_Details($expand=Product($expand=Category))");
request.parameters.set("$skiptoken", param.skiptoken);
client.request(request, destination);
```

So, if in the example there are no instances of TupelList and the official SAP docs say TupelList does not have a public constructor that accepts any parameters, what do you mean we don't know what customers are going to pass to the constructor? They shouldn't even know about it.

And please, do not resolve ongoing conversations as I go through each one to check if the appropriate changes are made. It's the responsibility of the initial commenter to resolve a conversation, not the other way around